common/types/block.go

```diff
@@ -105,8 +111,18 @@ func (id NodeID) ShortString() string {
 	return Shorten(name, 5)
 }
 
+// BytesToNodeID deserializes a byte slice into a NodeID
+func BytesToNodeID(b []byte) NodeID {
```

Comment:
I think that would be better left for the PR on resolving the NodeID type, because currently a lot of the tests use arbitrary length VRF keys so the length of a NodeID isn't fixed in the tests. What do you think?